scss/partials/_blog.scss

```diff
@@ -0,0 +1,398 @@
+// VT Blog Post Hero
+.blog-post-hero {
+    margin: 0;
+    padding-bottom: 0;
+    width: 42.5rem;
+
+
+    @include breakpoint(desktop) {
+        margin: 0 auto;
+        width: 100%;
+    }
+
+    @media only screen and (min-width: 866px) and (max-width: 1160px) {
+        // TODO discuss with Mark and the Web team about how to handle custom breakpoints
+        margin: 0 auto;
+        width: 100%;
+    }
+
+    &__title {
+        @include breakpoint(desktop) {
+            font-size: 8.4rem;
+        }
+
+        font-size: 4.2rem;
+    }
+
+    &__blog-author {
+        align-items: center;
+        display: grid;
+        grid-template-columns: 9rem 1fr;
+        padding-bottom: 7.5rem;
+        padding-top: 3.5rem;
+
+        &-img-container {
+            padding-right: 2rem;
+
+            &__author-image {
+                background: none no-repeat center center / cover;
+                width: 6rem;
+                height: 6rem;
+                border-radius: 50%;
+            }
+        }
+
+        &-info {
+            display: inline-block;
+
+            &-title {
+                color: $secondary-green-2;
+                font-size: 1.6rem;
+            }
+
+            &-name {
+                @extend .tk-degular-text;
+                font-size: 2rem;
+            }
+        }
+    }
+
+    &__time-social {
+        border-top: 0.1rem solid $cool-neutral-2;
+        display: grid;
+        grid-template-columns: 18rem 1fr;
+
+        &-read-time {
+            border-right: 0.1rem solid $cool-neutral-2;
+            padding-top: 1.7rem;
+
+            &--p-text,
+            &--p-time {
+                display: inline-block;
+                font-size: 1.8rem;
+            }
+
+            &--p-time {
+                @extend .tk-degular-text;
+                color: $secondary-green-2;
+                padding-right: 3.6rem;
+            }
+        }
+
+        &-links {
+            margin-left: auto;
+            margin-right: 0;
+            padding-top: 2.1rem;
+
+
+            .social-icon__link {
+                margin-right: 1.7rem;
+
+                &:last-child {
+                    margin-right: 0;
+                }
+
+                img {
+                    background-color: $secondary-green-2;
+                }
+
+            }
+        }
+
+    }
+}
+
+// Blog Article Post
+.blog-article {
+    background: rgba(5, 28, 44, 1);
+    background: linear-gradient(180deg, rgba(5, 28, 44, 1) 24.6rem, rgba(255, 255, 255, 1) 24.6rem);
+    padding-top: 3rem;
+    padding-bottom: 0;
+
+
+    &__image {
+        padding-top: 3.8rem;
+        padding-bottom: 6.6rem;
+
+        &-featured {
+            padding-bottom: 13.8rem;
+        }
+    }
+
+    &__post {
+        @include breakpoint(desktop) {
+            margin: 0 19.5rem;
+        }
+
+
```

Comment:
Breaking BEM here because I don't think content editors will remember or take the time to add class names for every header and paragraph tag.